pkg/render/monitor/monitor_test.go

```diff
@@ -154,6 +156,9 @@ var _ = Describe("monitor rendering tests", func() {
 		Expect(component.ResolveImages(nil)).NotTo(HaveOccurred())
 		toCreate, _ := component.Objects()
 
+		// Prometheus Operator
```

Comment:
I suppose this needs to be implemented still?

Reply:
Implemented.